permission-flow/src/main/java/dev/shreyaspatil/permissionFlow/initializer/PermissionFlowInitializer.kt

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2022 Shreyas Patil
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.shreyaspatil.permissionFlow.initializer
+
+import android.content.Context
+import androidx.startup.Initializer
+import dev.shreyaspatil.permissionFlow.PermissionFlow
+import dev.shreyaspatil.permissionFlow.impl.PermissionFlowImpl
+
+/**
+ * Initializes [PermissionFlow] instance on app startup.
+ */
+class PermissionFlowInitializer : Initializer<Unit> {
+
+    override fun create(context: Context) {
+        PermissionFlowImpl.init(context, PermissionFlow.DEFAULT_DISPATCHER)
```

Comment:
This method is overloaded already, no need to provide a default dispatcher already. 

```suggestion
        PermissionFlowImpl.init(context)
```

Reply:
Done.